package.json

```diff
@@ -129,27 +112,63 @@
     "@wordpress/url": "^2.9.0",
     "accessible-autocomplete": "^1.6.2",
     "autoprefixer": "^9.5.0",
+    "classnames": "^2.2.6",
+    "dompurify": "^2.0.8",
+    "element-closest": "^2.0.2",
+    "focus-trap-react": "^5.0.1",
+    "formdata-polyfill": "^3.0.17",
+    "intl": "^1.2.5",
+    "intl-locales-supported": "^1.6.0",
+    "invariant": "^2.2.4",
+    "jquery": "^3.4.1",
+    "lodash": "^4.17.15",
+    "md5": "^2.2.1",
+    "natives": "^1.1.6",
+    "polyfill-library": "^3.31.1",
+    "promise-polyfill": "^8.0.0",
+    "prop-types": "^15.7.2",
+    "psl": "^1.4.0",
+    "punycode": "^2.1.1",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "svgxuse": "^1.2.6",
+    "tinymce": "^5.1.6",
+    "url-polyfill": "^1.1.5",
+    "whatwg-fetch": "^3.0.0"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.5.0",
+    "@babel/core": "^7.5.4",
+    "@babel/plugin-syntax-dynamic-import": "^7.2",
+    "@babel/plugin-transform-runtime": "^7.6.0",
+    "@babel/polyfill": "^7.6.0",
+    "@babel/preset-env": "^7.6.0",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/register": "^7.4.4",
+    "@babel/traverse": "^7.5.5",
+    "@storybook/addon-viewport": "^5.0.11",
+    "@storybook/react": "^5.0.11",
+    "@testing-library/jest-dom": "^4.1.2",
+    "@testing-library/react": "^9.4.1",
+    "@wordpress/babel-preset-default": "^4.5.0",
+    "@wordpress/browserslist-config": "^2.6.0",
+    "@wordpress/e2e-test-utils": "^2.1.0",
+    "@wordpress/eslint-plugin": "^4.0.0",
     "babel-eslint": "^10.0.3",
     "babel-loader": "^8.0.5",
     "babel-plugin-lodash": "^3.3.4",
     "backstopjs": "3.8.8",
     "browser-sync": "^2.26.3",
     "bundlesize": "^0.17.1",
-    "circular-dependency-plugin": "^5.2.0",
```

Comment:
@ShahAaron tests are failing because this was removed but webpack is configured to use it. The diff shows +45, -44 so that may be the only one. There should be the same number of lines changed +2 for the added `dependencies` key and an added closing `},`.

Please double check that there are no others missing 😄 

PS - `circular-dependency-plugin` is a `devDependency` 😉

Reply:
https://travis-ci.com/github/google/site-kit-wp/jobs/323769284#L1310